cluster/gce/config-default.sh

```diff
@@ -95,6 +95,12 @@ MASTER_EXTRA_METADATA=${KUBE_MASTER_EXTRA_METADATA:-${KUBE_EXTRA_METADATA:-}}
 NODE_EXTRA_METADATA=${KUBE_NODE_EXTRA_METADATA:-${KUBE_EXTRA_METADATA:-}}
 # KUBELET_TEST_ARGS are extra arguments passed to kubelet.
 KUBELET_TEST_ARGS=${KUBE_KUBELET_EXTRA_ARGS:-}
+if [[ "${CONTAINER_RUNTIME}" == "containerd" ]]; then
+  CONTAINER_RUNTIME_NAME=${KUBE_CONTAINER_RUNTIME_NAME:-containerd}
+  CONTAINER_RUNTIME_ENDPOINT=${KUBE_CONTAINER_RUNTIME_ENDPOINT:-unix:///run/containerd/containerd.sock}
+  LOAD_IMAGE_COMMAND=${KUBE_LOAD_IMAGE_COMMAND:-ctr -n=k8s.io images import}
+  KUBELET_TEST_ARGS="${KUBE_KUBELET_EXTRA_ARGS:-} --runtime-cgroups=/system.slice/containerd.service"
```

Comment:
How about appending to `KUBELET_TEST_ARGS` directly? That way if line 97 changes, this wouldn't break

Reply:
Done